help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+---
+title: (Beta) The Trade Desk - CRM connection
```

Comment:
I also notice that you uploaded some files which are not referenced in the documentation at all (e.g. `catalogdestination1.png`). Please remove any files that you are not planning to link to from the documentation.

Reply:
Updated.